examples/misc/lib/effective_dart/usage_good.dart

```diff
@@ -180,7 +181,7 @@ void miscDeclAnalyzedButNotTested() {
   {
     // ignore_for_file: only_throw_errors
     // #docregion async
-    Future<void> usesAwait(Future<void> later) async {
+    Future<void> usesAwait(Future<Object> later) async {
```

Comment:
nit: I don't think `<Object>` is required here, this could be changed to `Future later`.

Reply:
@munificent what do you think? Clearly, `Future <void>` was the wrong type to use. :)

(I'm going to merge now, so the build is fixed, but I can easily change this later.)

Reply:
Yeah, `void` is definitely wrong. `String` would probably be clear and simple since the result gets passed to `print()`.